lib/index.js

```diff
@@ -46,10 +47,13 @@ export default class extends Component {
     this.offset = width - height + 1
     this.handlerSize = height - 2
 
+    const start = I18nManager.isRTL ? 'flex-end' : 'flex-start';
```

Comment:
You'd better place `start` and `stop` in `this` of constructor, like following: 
```javascript
constructor () {
  this.start = xxxx
  this.end = xxxx // please rename 'stop' as 'end'
}
``` 
It's just a relatively static value and it will be assigned when the component initializes and won't change in lifecycle of the component.